build.gradle

```diff
@@ -401,6 +382,9 @@ class PreDist extends DefaultTask {
             filter { String line -> line.replace('${build.timestamp}', buildTimestamp) }
             into 'build/generated-dist/data/resourcemanager'
         }
+        project.delete(
```

Comment:
Should not be necessary and will likely cause Gradle task caching to not work. If this was added to address a build issue, let's create a separate issue for it and resolve it separately from this feature implementation.

Reply:
I added because the distribution.zip seems contain ${group} folder with plugin.xml (from template) as well as gov.nasa.jpl.cae.magicdraw.mdk folder.  But removed for now.  I just committed and pushed.

Reply:
See https://github.com/Open-MBEE/mdk/pull/207#discussion_r678726924.